projects/public-search/src/app/document-detail/holdings/holding/holding.component.html

```diff
@@ -109,7 +100,12 @@
     </ng-container>
   </dl>
 </div>
-<public-search-items [holding]="holding"
-                     [viewcode]="viewcode"
-                     [hidden]="collapsed || !collapsable"
-></public-search-items>
+<public-search-items *ngIf="holding.metadata.public_items_count > 0" [holding]="holding" [viewcode]="viewcode" [hidden]="collapsed || !collapsable"></public-search-items>
+<public-search-request
+  [record]="holding"
+  recordType="holding"
+  [viewcode]="viewcode"
+  [hidden]="collapsed || !collapsable"
```

Comment:
You can put a div or ng-container around `public-search-items` and `public-search-request` to add an `*ngIf="!collapse"` condition and remove it from the children.